transcription/transcription.go

```diff
@@ -38,6 +45,49 @@ func msgHeaders(from string, to []string, subject string) string {
 	return strings.Join(msgHeaders, "\r\n")
 }
 
+// StartTranscription transcribes a given file using Sphinx.
+// File name should be in "name.wav" format.
+func StartTranscription(fileName string, command string) (string, error) {
+
+	cmd := exec.Command("java", "-jar", command, fileName)
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	outputFileName := "/Sphinx/files/" + fileName + "-json.txt"
+
+	// //once json is sent somewhere, capture the output
+	// if _, err := transcriptionOutputToStruct(outputFileName); err != nil {
+	// 	return err
+	// }
+	return outputFileName, nil
```

Comment:
The transcription output is a file which is stored at that filename location - maybe it would make sense to rename the variable to 'outputFileLoc' or 'outputFile'? The second method (transcriptionOutputToStruct) reads the output file and creates a struct. One option would be for this method to call transcriptionOutputToStruct and return a struct. I think that makes a lot of sense.